### DISCUSSION TITLE
Resize output to source

### DISCUSSION THREAD

**Discussion by pjhfggij** [obsproject/obs-studio#10813] [Development Questions]
[As described peviously](https://obsproject.com/forum/threads/resize-base-and-output-res-to-current-source-size.138795/) capturing only the area of a source without the rest of the output is very cumbersome in obs right now. Does anyone know any script to output video only in resolution of the selected region?

The desired result:
![image](https://github.com/obsproject/obs-studio/assets/161205138/c4738135-e948-4262-8e5c-73ee3f6934dd)

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!

**Reply by pjhfggij** [reply]
No worries, I [found out](https://github.com/obsproject/obs-studio/issues/10814) that this is not possible currently and created a [new idea](https://ideas.obsproject.com/posts/2609/resize-output-source-size-to-source-crop) to solve it.

If anyone comes across it from the future - you can add a "Crop/Pad Filter" to this source, which will then make "Resize output (source size)" work - it is cumbersome, but that's the only way I know to make this work right now.
